renderer.go

```diff
@@ -23,6 +24,41 @@ type formats struct {
 	clearOEL    string
 }
 
+var (
+	// Map for color names and their respective foreground [0] or background [1] color codes
+	ColorMap map[string][2]string = map[string][2]string {
```

Comment:
Went for a map in order to still have one key identifying a background-foreground color pair and not get too complex `if`s and `switch`es. What do you think? 🤔